packages/launchpad/src/launchpad.tsx

```diff
@@ -89,7 +90,7 @@ const Launchpad = ( {
 			siteSlug,
 			tracksData,
 			extraActions: {
-				setActiveChecklist,
+				...( config.isEnabled( 'launchpad/navigator' ) ? { setActiveChecklist } : {} ),
```

Comment:
It also feels like we should handle this within `setupActionsForTasks()` rather than in callers. I'll also fold that in as part of an update.